src/test/app/build_test_jshooks.sh

```diff
@@ -15,18 +39,18 @@ namespace ripple {
 namespace test {
 std::map<std::string, std::vector<uint8_t>> jswasm = {' > $OUTPUT_FILE
 COUNTER="0"
-cat SetJSHook_test.cpp | tr '\n' '\f' |
-        ggrep -Po 'R"\[test\.hook\](.*?)\[test\.hook\]"' | 
-        gsed -E 's/R"\[test\.hook\]\(//g' | 
-        gsed -E 's/\)\[test\.hook\]"[\f \t]*/\/*end*\//g' | 
+cat "$INPUT_FILE" | tr '\n' '\f' |
+        grep -Po 'R"\[test\.hook\](.*?)\[test\.hook\]"' | 
```

Comment:
Why not just use ggrep ? 
If it's targeting gnu grep/sed specifically ?

> brew install gnu-sed grep

This installs the gsed/ggrep commands, no?

Reply:
This script should be usable in both macos and ubuntu environments.
Therefore, scripts that are only usable in macos (gsed/ggrep) should not be used here.


you can use gsed/ggrep as sed/grep by add PATH.
```
add path: PATH="/opt/homebrew/opt/gnu-sed/libexec/gnubin:$PATH"
```

Reply:
Oh, I didn't realize that ggrep and gsed weren't available on Ubuntu

It's been an embarrassingly long time since I've used linux/ubuntu